specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ContainerApps.json

```diff
@@ -551,6 +615,129 @@
           "type": "boolean"
         }
       }
+    },
+    "CustomDomain": {
+      "description": "Custom Domain of a Container App",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Hostname.",
+          "type": "string"
+        },
+        "bindingType": {
+          "description": "Custom Domain binding type.",
+          "enum": [
+            "Disabled",
```

Comment:
>Disabled [](http://example.com/codeflow?start=13&length=8)

if user sets this to disabled and allowInsecure to false - I assume the API call still succeeds but the app won't work?

Reply:
Correct. It won't work for that specific host name. I don't think we should fail the API call as this could be used as a way for users to quickly "bring offline" their app.